Sources/CAudioKit/Nodes/Generators/Physical Models/ClarinetDSP.mm

```diff
@@ -1,6 +1,10 @@
 // Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
 
-#import "STKInstrumentDSP.hpp"
+#if __APPLE__
+#include "STKInstrumentDSP.hpp"
+#else // __APPLE__
+#include "STKInstrumentDSP.hpp"
+#endif // __APPLE__
```

Comment:
?

Reply:
Hi! Thanks for the feedback. Yep, well spotted (this and the above)! I'm guessing those are easy for you to change after doing a merge? Not really sure how handling merge requests issues work yet!

NB: I have another set of improvements that I'll pull together as a separate pull request, once these are merged, that I think you'll find useful, as I've experimented more on iOS/macOS/tvOS/WIndows/Android (I write cross-platform audio apps).

Best wishes, Pete

Reply:
That includes some example cross-platform C++ code I'll add to the PORTING file.

Reply:
FYI you can just keep pushing commits to your branch and they will show up in this pull request.

Reply:
@megastep Aha - many thanks, I didn't actually know that :)

Pete

Reply:
Fixed and pushed!